### ISSUE TITLE
HTTPError: HTTP Error 401: The request's Authentication (Server-) Key contained an invalid or malformed FCM-Token (a.k.a . IID-Token)

### ISSUE THREAD

**Issue by akamanzi** [jazzband/django-push-notifications#452]
I am getting the above error, I am not sure but i think its something related to authorisation with Firebase. I tried different options and search to no success. does anyone know how to deal with this.

I got this error while trying to send a message using GCMDevice `device.send_message`.
Any help is highly appreciated

**Comment by jamaalscarlett** [comment]
@akamanzi Are you using the correct api key?  Are you able to send messages to the device using the firebase console?

**Comment by akamanzi** [comment]
@jamaalscarlett Yes, i used the correct API key and the notification is sent on firebase console. Am not sure what the issue is

**Comment by jamaalscarlett** [comment]
If you are seeing the error when using the firebase console, the problem is not with this package.  Perhaps the device's registration id is incorrect

**Comment by kruglyjmax** [comment]
@akamanzi Hi, try new server key. [https://stackoverflow.com/a/37801206](https://stackoverflow.com/a/37801206) Helped for me.
